frontend/src/libs/components/notification/notification.tsx

```diff
@@ -0,0 +1,10 @@
+import 'react-toastify/dist/ReactToastify.css';
```

Comment:
do we need this import here?

Reply:
should styles be imported within the component or at the time the component is used, and which approach is better?